src/styles/defaultTheme.js

```diff
@@ -28,8 +28,9 @@ export default {
         warning: COLOR_WARNING,
         background: {
             primary: COLOR_WHITE,
-            secondary: COLOR_GRAY_2,
+            secondary: '#f9fafc;',
         },
+        divider: COLOR_GRAY_2,
```

Comment:
For now, let's not use this, @TahimiLeonBravo changed where it is used and set a rgba color that looks good with every theme. Maybe in the future, we define this in the theme API but we will not use it yet.